.github/workflows/release.yaml

```diff
@@ -52,23 +52,10 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
 
-      - name: "Get tag name"
```

Comment:
should be a lot more confident on releases with less shell scripts in yaml! cc @llinder